### PR TITLE
Update bookings-faq.md

### PR DESCRIPTION
**Pull request by msbemba**
Changed entire document to reflect new name changes with M365 business suite and also modified availability to reflect availability on M365 Business Standard as well..

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Isuse#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3562

File: microsoft-365/bookings/bookings-faq.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Yes. We provide a link to your booking page within the **Booking page** tab in t
 
 ### Can our IT department control whether end users can access Bookings?
 
-Bookings is available in Business Premium, M365 A3/A5 and E3/E5 tenants by default, but administrators can turn it off in the Microsoft 365 admin center if they choose. To do so, [follow these instructions](turn-bookings-on-or-off.md).
+Bookings is available in Microsoft 365 Business Premium, Microsoft 365 Business Standard, M365 A3/A5 and E3/E5 tenants by default, but administrators can turn it off in the Microsoft 365 admin center if they choose. To do so, [follow these instructions](turn-bookings-on-or-off.md).
 
 If a customer wants to disable access to Bookings for certain eligible licenses in their tenant, they can either user a group policy to restrict licenses or implement an OWA Mailbox policy that will restrict who is able to create new Bookings calendars.
```
Comment (JohanFreelancer9) [inline microsoft-365/bookings/bookings-faq.md]:
```suggestion
If a customer wants to disable access to Bookings for certain eligible licenses in their tenant, they can either use a group policy to restrict licenses or implement an OWA Mailbox policy that will restrict who is able to create new Bookings calendars.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - one additional edit, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**